app/src/main/java/fi/helsinki/opintoni/integration/feed/FeedRemoteClient.java

```diff
@@ -19,11 +19,18 @@
 
 import com.rometools.fetcher.FeedFetcher;
 import com.rometools.rome.feed.synd.SyndFeed;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.net.URL;
 import java.util.Optional;
 
 public class FeedRemoteClient implements FeedClient {
+
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FeedRemoteClient.class);
+
+
```

Comment:
Tarvitaanko näin paljon tyhjiä rivejä? Olisi hyvä jos kaksi peräkkäistä tyhjää riviä saisi kiellettyä konffissa.

Reply:
Lisäsin säännöt: 
 <property name="allowMultipleEmptyLinesInsideClassMembers" value="false"/>
  <property name="allowMultipleEmptyLines" value="false"/>

Nämä voisi lisätä UniHow-puolellekin?

Reply:
Joo, voisi ottaa samat lisäykset UniHow:n survey-servicellekin. Tee vaan PR heti kun ehdit.